### ISSUE TITLE
Is there a plan to support 3D model tile formats such as 3DTiles and I3S?

### ISSUE THREAD

**Issue by xrui94** [OSGeo/gdal#11538]
### Feature description

Cesium 3DTiles and ESRI I3S are commonly used data formats in 3D GIS and have long been included in OGC standards. I think gdal should support them.

**Comment by rouault** [comment]
> Is there a plan to support 3D model tile formats such as 3DTiles and I3S?

No, GDAL lacks support for "real" 3D in its abstraction model. GDAL can support 3D geometries, but for proper 3D there are other properties like texturing that should be added. So basically that would require adding another abstraction model besides the raster and vector ones. 
https://github.com/assimp/assimp is probably the closest to a "GDAL for 3D" that I'm aware of
